Makefile

```
@@ -2,7 +2,11 @@ RELEASE_DIR:=out
 MMDC := ./node_modules/.bin/mmdc
 DIAGRAMS := $(patsubst %.mmd,%.svg,$(wildcard specs/diagrams/*.mmd))
 
-build: specs/index.html specs/faq.html
+AZURE_STORAGE_ACCOUNT := ghpreview
+AZURE_STORAGE_CONTAINER := preview
+
+
+build: specs/index.html
```

Is the omission of `specs/faq.html` purposeful? 
if not:
```suggestion
build: specs/index.html specs/faq.html
```